infra/data-pipeline.yml

```diff
@@ -444,6 +444,46 @@ tasks:
         owner: release-mgmt-analysis@mozilla.com
         source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
 
+    - ID: check_metrics
+      created: {$fromNow: ''}
+      deadline: {$fromNow: '119 hours'}
+      provisionerId: aws-provisioner-v1
+      workerType: releng-svg
+      dependencies:
+        - train-backout
+        - train-component
+        - train-defectenhancementtask
+        - train-duplicate
+        - train-regression
+        - train-regressionrange
+        - train-regressor
+        - train-stepstoreproduce
+        - train-tracking
+      payload:
+        maxRunTime: 3600
+        image: python
+        command:
+          - "/bin/bash"
+          - "-lcx"
+          - "git clone --quiet https://github.com/mozilla/bugbug /code &&
+             cd /code &&
+             git checkout ${version} &&
+             pip install --quiet . &&
+             pip install --quiet -r test-requirements.txt &&
```

Comment:
Can we use the bugbug base image instead of cloning bugbug and installing requirements?

Reply:
It should works but we may not ship the scripts directory in a very friendly manner. We could also add the three scripts as console scripts but we already have 7 console scripts this might be a bit too much.

Reply:
Yeah, we have many... But I guess it's fine, I wouldn't worry too much about that